### PR TITLE
chore: Add integration test for AWSBedrockRuntime's InvokeModelWithBidirectionalStream

### PR DESCRIPTION
**Pull request by sichanyoo**
## Issue \#

N/A

## Description of changes

- Adds an integration test for [AWSBedrockRuntime's InvokeModelWithBidirectionalStream API](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_InvokeModelWithBidirectionalStream.html) that released last week.

## Conventional Commits

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/InvokeModelWithBidirectionalStreamTest.swift [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        let resp = try await bedrock.invokeModelWithBidirectionalStream(input: InvokeModelWithBidirectionalStreamInput(
+            body: inputStream,
+            modelId: modelID
+        ))
+
+        // Iterate on the returned output stream for output events.
+        for try await output in resp.body! {
+            switch output {
+            case .chunk(let output):
+                XCTAssertTrue(output.bytes != nil && !output.bytes!.isEmpty)
```
Comment (jbelkins) [inline IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/InvokeModelWithBidirectionalStreamTest.swift:227]: is there anything more that we can verify for test success in the response stream, or is the fact that it contains non-zero audio data the best we can do?

Comment (jbelkins) [reply IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/InvokeModelWithBidirectionalStreamTest.swift:227]: (i.e. in the transcribe tests we check that the transcription streamed back matches what we expected from the audio we sent)

**Review by dayaffe — APPROVED** [review]

**Review by jbelkins — COMMENTED** [review]
One general question, otherwise good.

File: IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/InvokeModelWithBidirectionalStreamTest.swift (hunk 1 above)
Comment (sichanyoo) [reply IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/InvokeModelWithBidirectionalStreamTest.swift:227]: Given the length and variability of what exactly the model says in response, I think asserting on byte being non-null might be the best we can do for now.

**Comment by sichanyoo** [comment]
Pending CRT investigation on why server complains about frames in requests and closes the connection prematurely.

**Review by jbelkins — APPROVED** [review]
